WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -89,6 +89,20 @@ + (void)disableScreenshots
   [[NSUserDefaults standardUserDefaults] setBool:YES forKey:@"DisableScreenshots"];
 }
 
++ (void)enableScreenshots
+{
+  [[NSUserDefaults standardUserDefaults] setBool:NO forKey:@"DisableScreenshots"];
+}
+
++ (void)disableScreenshotsUnlessEnvironment
```

Comment:
I think it makes sense to inline this logic into UITestingUITests, because it is only used there

Reply:
right! thanks